dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```diff
@@ -166,20 +166,31 @@ public Response toolsIndexGet(SecurityContext securityContext) {
                     .builder(new HttpHost(config.getEsConfiguration().getHostname(), config.getEsConfiguration().getPort(), "http"))
                     .build()) {
 
-                // Delete index
+                // Delete indices
                 try {
-                    restClient.performRequest("DELETE", "/entry");
```

Comment:
Same with above, should make deleting tools and deleting workflows a re-usable method in case we want to add new indices some day (perhaps notebooks or services)